pkg/core/native/native_test/neo_test.go

```diff
@@ -900,3 +901,88 @@ func TestNEO_GetCandidates(t *testing.T) {
 	neoCommitteeInvoker.Invoke(t, expected, "getCandidates")
 	checkGetAllCandidates(t, expected)
 }
+
+func TestNEO_RegisterViaNEP27(t *testing.T) {
+	const echidnaHeight = 12 // Same as Domovoi in UT config.
```

Comment:
I'd suggest to avoid custom modifications of unittestnet configuration as much as possible. Instead, we may directly set Echidna height in the `./config/protocol.unit_testnet.yml` to some 14 and use this value from unit tests. We did https://github.com/nspcc-dev/neo-go/pull/3696 specifically to reuse embedded config everywhere.

Reply:
But of course it's not a big deal if you'd like to keep this code. I just want unittest config related things to be unified.

Reply:
My initial attempts at enabling it everywhere broke unrelated tests. I'll check one more time.

Reply:
I think it's fixed now.